### ISSUE TITLE
no more mantained

### ISSUE THREAD

**Issue by roccomuso** [dustmoo/gridster.js#51]
Module no more mantained right?
we should switch to: https://github.com/troolee/gridstack.js

**Comment by dustmoo** [comment]
Yes this hasn't been maintained in ages, mainly due to poor base-architecture and a decided lack of time to redesign it.
